openvdb/openvdb/tree/ValueAccessor.h

```diff
@@ -1493,16 +1909,15 @@ class ValueAccessor1 : public ValueAccessorBase<_TreeType, IsSafe>
     const NodeT* probeConstNode(const Coord& xyz) const
     {
         assert(BaseT::mTree);
-        OPENVDB_NO_UNREACHABLE_CODE_WARNING_BEGIN
         if ((std::is_same<NodeT, NodeT0>::value)) {
             if (this->isHashed(xyz)) {
                 assert(mNode0);
                 return reinterpret_cast<const NodeT*>(mNode0);
             }
             return BaseT::mTree->root().template probeConstNodeAndCache<NodeT>(xyz, this->self());
         }
-        return nullptr;
-        OPENVDB_NO_UNREACHABLE_CODE_WARNING_END
+        // No point trying to cache as this NodeT isn't part of this accessors chain
+        return BaseT::mTree->root().template probeConstNode<NodeT>(xyz);
```

Comment:
Bugfix - even if `NodeT` isn't being cached it might exist and a nullptr will always be returned